package.json

```diff
@@ -102,7 +102,16 @@
       "allowedVersions": {
         "@octokit/core": ">=3"
       }
-    }
+    },
+    "onlyBuiltDependencies": [
+      "@swc/core",
+      "better-sqlite3",
+      "esbuild",
+      "sharp",
+      "sqlite3",
+      "workerd",
+      "yarn"
+    ]
```

Comment:
I believe this should be in `pnpm-workspace.yaml` instead of `package.json` with pnpm 10: https://pnpm.io/settings#onlybuiltdependencies